### ISSUE TITLE
Issue with MGB-3 FInetuning from MGB2-Recipe

### ISSUE THREAD

**Issue by khelfiakram1** [espnet/espnet#6051]
Hello, I'm currently doing some experiments on the MGB-3 data, i pretrained the model on MGB-2 dataset following the recipe under egs/mgb2/asr1 and when finetuning i followed the paper (They reduced learning rate to 0.1 with no warmup steps) and i'am getting those results. 
These are the results : 

![Image](https://github.com/user-attachments/assets/4291ee0b-b93d-41b1-b4d1-b3f34d71d129)
![Image](https://github.com/user-attachments/assets/7133fb2d-e509-4666-badb-6f1366cbe693)

This is the config i am using
`# This configuration requires 4 gpus with 12GB memory
accum-grad: 4
adim: 512
aheads: 8
backend: pytorch
batch-bins: 5000000
dlayers: 6
dropout-rate: 0.1
dunits: 2048
elayers: 12
epochs: 30
eunits: 2048
grad-clip: 5
lsm-weight: 0.1
model-module: espnet.nets.pytorch_backend.e2e_asr_transformer:E2E
mtlalpha: 0.3
opt: noam
patience: 0
sortagrad: 0
transformer-attn-dropout-rate: 0.0
transformer-init: pytorch
transformer-input-layer: conv2d
transformer-length-normalized-loss: false
transformer-lr: 0.1
transformer-warmup-steps: 1
enc-init: /espnet/egs/mgb2/asr1/exp/train_trim_sp_pytorch_2gpu_2grad/results/model.val5.avg.best
dec-init: /egs/mgb2/asr1/exp/train_trim_sp_pytorch_2gpu_2grad/results/model.val5.avg.best
enc-init-mods: 'encoder.'
dec-init-mods: 'decoder.'`

**Comment by sw005320** [comment]
I think you need some learning rate tuning.
The configuration depends on many factors (GPU type, Pytorch/cuda/cudnn version, etc.).
It is difficult to reproduce the same results.

**Comment by khelfiakram1** [comment]
But from the curves we see that the model is not even learning, i tried to lower the learning rates and the results got worse

**Comment by sw005320** [comment]
Yeah, this is what I meant to say. I recommend you tweak several optimization hyper-parameters, including learning rate and batch size, to make it work.

**Comment by khelfiakram1** [comment]
i experimented with different learning rates and batch size and without any improvement

**Comment by khelfiakram1** [comment]
I tried different LR but this is 2 i choose :
For warmup in the paper they said no warmup so i set it to 1 is that correct ? 
 I'm i on a good range ? 

this is with LR: 0.001 

![Image](https://github.com/user-attachments/assets/62ce9d75-7678-4436-b148-9362a120bfbb)

and this with LR: 0.1

![Image](https://github.com/user-attachments/assets/b39c090f-baff-43b8-a76c-26b60de263a3)
